setup.py

```diff
@@ -73,5 +78,13 @@ def _get_version():
         'coverage': RunCoverage,
         'test': RunTests,
     },
-    include_package_data=True
+    include_package_data=True,
+    entry_points={
+        'console_scripts': [
+            'bibliopixel = bibliopixel.main.main:main'
+        ]
+    },
+    install_requires=[
+        'BiblioPixelAnimations'
```

Comment:
So it knows that this is your BPA because - ah, because you have installed it on pypi.  Got it.  

Looks good to me!

:+1: